### PR TITLE
update ovh-ui-kit and ovh-ui-angular to v2.0.0

### PR DESCRIPTION
**Pull request by virgotopia**

### CODE REVIEW

File: client/app/changelog/changelog.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div data-ng-controller="ChangelogCtrl as Changelog">
     <div class="modal-header"></div>
     <div class="modal-body">
-        <oui-loader data-ng-if="!Changelog.content"
-                    data-size="m">
```
Comment (AxelPeter) [inline client/app/changelog/changelog.html]: (optionnal) We can remove the attribute `data-size="m"` on all `<oui-spinner>`. It's default value of the attribute (see http://master.ui-kit.ovh/#!/oui-angular/spinner)

File: client/app/telecom/dashboard/bills/telecom-dashboard-bills.html [hunk 2]
```diff
@@ -72,7 +72,7 @@
             <tbody data-ng-if="BillsCtrl.loaders.bills">
                 <tr>
                     <td colspan="4">
```
Comment (Jisay) [inline client/app/telecom/dashboard/bills/telecom-dashboard-bills.html]: \+ `text-center`

File: client/app/telecom/freeFax/information/freeFax-information.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
     </h3>
     <div class="oui-tile__body"
```
Comment (Jisay) [inline client/app/telecom/freeFax/information/freeFax-information.html]: \+ `text-center`

File: client/app/telecom/freeFax/notifications/freeFax-notifications.html [hunk 4]
```diff
@@ -49,7 +49,7 @@
                     </tr>
                     <tr data-ng-hide="FreeFaxNotifications.notifications">
                         <td colspan="4">
```
Comment (Jisay) [inline client/app/telecom/freeFax/notifications/freeFax-notifications.html]: \+ `class="text-center"`

File: client/app/telecom/orders/telecom-orders.html [hunk 5]
```diff
@@ -57,7 +57,7 @@ <h1 data-translate="telecom_orders_follow_up_title"></h1>
                         </tr>
                         <tr data-ng-show="OrdersCtrl.loading">
                             <td colspan="5">
```
Comment (Jisay) [inline client/app/telecom/orders/telecom-orders.html]: \+ `class="text-center"`

File: client/app/telecom/overTheBox/details/overTheBox-details.html [hunk 6]
```diff
@@ -222,7 +221,7 @@
                         </h3>
                         <div class="oui-tile__body"
```
Comment (Jisay) [inline client/app/telecom/overTheBox/details/overTheBox-details.html]: \+ `text-center`

File: client/app/telecom/overTheBox/details/overTheBox-details.html [hunk 7]
```diff
@@ -256,7 +255,7 @@
...
```
Comment (Jisay) [inline client/app/telecom/overTheBox/details/overTheBox-details.html]: \+ `text-center`

File: client/app/telecom/overTheBox/remote/overTheBox-remote.html [hunk 8]
```diff
@@ -27,7 +27,7 @@
                     <tbody data-ng-if="OverTheBoxRemote.remotes.length === 0">
                         <tr data-ng-show="OverTheBoxRemote.loading">
                             <td colspan="5">
```
Comment (Jisay) [inline client/app/telecom/overTheBox/remote/overTheBox-remote.html]: \+ `text-center`

File: client/app/telecom/overTheBox/tasks/overTheBox-tasks.html [hunk 9]
```diff
@@ -32,7 +32,7 @@
                     <tbody data-ng-if="OverTheBoxTasks.loaders.init">
                         <tr>
                             <td colspan="3">
```
Comment (Jisay) [inline client/app/telecom/overTheBox/tasks/overTheBox-tasks.html]: \+ `class="text-center"`

File: client/app/telecom/pack/slots/domain/pack-domain.html [hunk 10]
```diff
@@ -2,7 +2,7 @@
     <div class="oui-tile__body"
          data-ng-if="loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/domain/pack-domain.html]: \+ `text-center`

File: client/app/telecom/pack/slots/exchangeAccount/pack-exchangeAccount.html [hunk 11]
```diff
@@ -2,7 +2,7 @@
     <div class="oui-tile__body"
          data-ng-if="loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/exchangeAccount/pack-exchangeAccount.html]: \+ `text-center`

File: client/app/telecom/pack/slots/exchangeIndividual/pack-exchangeIndividual.html [hunk 12]
```diff
@@ -2,7 +2,7 @@
     <div class="oui-tile__body"
          data-ng-if="loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/exchangeIndividual/pack-exchangeIndividual.html]: \+ `text-center`

File: client/app/telecom/pack/slots/exchangeLite/pack-exchangeLite.html [hunk 13]
```diff
@@ -3,7 +3,7 @@
     <div class="oui-tile__body"
          data-ng-if="loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/exchangeLite/pack-exchangeLite.html]: \+ `text-center`

File: client/app/telecom/pack/slots/hostedEmail/pack-hostedEmail.html [hunk 14]
```diff
@@ -3,7 +3,7 @@
     <div class="oui-tile__body"
          data-ng-if="HostedEmail.loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/hostedEmail/pack-hostedEmail.html]: \+ `text-center`

File: client/app/telecom/pack/slots/hubic/pack-hubic.html [hunk 15]
```diff
@@ -2,7 +2,7 @@
     <div class="oui-tile__body"
          data-ng-if="Hubic.loaders.service">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/hubic/pack-hubic.html]: \+ `text-center`

File: client/app/telecom/pack/slots/voipBillingAccount/pack-voipBillingAccount.html [hunk 16]
```diff
@@ -2,7 +2,7 @@
     <div class="oui-tile__body"
          data-ng-if="loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/voipBillingAccount/pack-voipBillingAccount.html]: \+ `text-center`

File: client/app/telecom/pack/slots/voipEcoFax/pack-voipEcoFax.html [hunk 17]
```diff
@@ -2,7 +2,7 @@
     <div class="oui-tile__body"
          data-ng-if="loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/voipEcoFax/pack-voipEcoFax.html]: \+ `text-center`

File: client/app/telecom/pack/slots/voipLine/pack-voipLine.html [hunk 18]
```diff
@@ -2,7 +2,7 @@
     <div class="oui-tile__body"
          data-ng-if="loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/voipLine/pack-voipLine.html]: \+ `text-center`

File: client/app/telecom/pack/slots/xdslAccess/pack-xdslAccess.html [hunk 19]
```diff
@@ -3,7 +3,7 @@
     <div class="oui-tile__body"
          data-ng-if="loaders.services">
         <div class="oui-tile__definition">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/xdslAccess/pack-xdslAccess.html]: \+ `text-center`

File: client/app/telecom/pack/xdsl/access/ip/pack-xdsl-access-ip.html [hunk 20]
```diff
@@ -133,7 +133,7 @@ <h2 class="widget-presentation-title">
                         </tr>
                         <tr data-ng-if="XdslAccessIp.loading">
                             <td colspan="3">
```
Comment (Jisay) [inline client/app/telecom/pack/xdsl/access/ip/pack-xdsl-access-ip.html]: \+ `class="text-center"`

File: client/app/telecom/pack/xdsl/modem/bridgeMode/pack-xdsl-modem-bridgeMode.html [hunk 21]
```diff
@@ -17,7 +17,7 @@
             <span data-translate="xdsl_modem_bridge_mode_status"></span>
         </label>
         <div data-ng-if="BridgeCtrl.mediator.tasks.changeModemConfigBridgeMode">
```
Comment (Jisay) [inline client/app/telecom/pack/xdsl/modem/bridgeMode/pack-xdsl-modem-bridgeMode.html]: \+ `text-center`

File: client/app/telecom/pack/xdsl/modem/dmz/pack-xdsl-modem-dmz.html [hunk 22]
```diff
@@ -51,7 +51,7 @@ <h2 class="widget-presentation-title" data-translate="xdsl_modem_dmz_status"></h
 									data-ui-validate="'DmzCtrl.validator.isIP($value)'"/>
                         </td>
                         <td data-ng-if="DmzCtrl.mediator.tasks.changeModemConfigDMZ">
```
Comment (Jisay) [inline client/app/telecom/pack/xdsl/modem/dmz/pack-xdsl-modem-dmz.html]: \+ `class="text-center"`

File: client/app/telecom/sms/dashboard/telecom-sms-dashboard.html [hunk 23]
```diff
@@ -67,7 +66,7 @@
                 </h3>
                 <div class="oui-tile__body"
```
Comment (Jisay) [inline client/app/telecom/sms/dashboard/telecom-sms-dashboard.html]: \+ `text-center`

File: client/app/telecom/sms/options/response/telecom-sms-options-response.html [hunk 24]
```diff
@@ -255,7 +254,7 @@ <h1 data-translate="sms_options_response"></h1>
                         <tbody data-ng-if="TelecomSmsOptionsResponseCtrl.loading.init">
                             <tr>
                                 <td colspan="3">
```
Comment (Jisay) [inline client/app/telecom/sms/options/response/telecom-sms-options-response.html]: \+ `class="text-center"`

File: client/app/telecom/sms/receivers/telecom-sms-receivers.html [hunk 25]
```diff
@@ -292,7 +290,7 @@
                     <tbody data-ng-if="SmsReceiversCtrl.receivers.isLoading">
                         <tr>
                             <td colspan="6">
```
Comment (Jisay) [inline client/app/telecom/sms/receivers/telecom-sms-receivers.html]: \+ `text-center`

File: client/app/telecom/sms/senders/blacklisted/telecom-sms-senders-blacklisted.html [hunk 26]
```diff
@@ -134,7 +133,7 @@ <h1 data-translate="sms_senders_blacklisted_manage_title"></h1>
                 <tbody data-ng-if="SmsSendersBlacklistedCtrl.blacklists.isLoading">
                     <tr>
                         <td colspan="4">
```
Comment (Jisay) [inline client/app/telecom/sms/senders/blacklisted/telecom-sms-senders-blacklisted.html]: \+ `text-center`

File: client/app/telecom/sms/sms/compose/addPhonebookContact/telecom-sms-sms-compose-addPhonebookContact.html [hunk 27]
```diff
@@ -108,7 +108,7 @@ <h3 data-translate="sms_sms_compose_add_phonebook_contact_title"></h3>
                     <tbody data-ng-if="AddPhonebookContactCtrl.phonebookContact.isLoading">
                         <tr>
                             <td colspan="5">
```
Comment (Jisay) [inline client/app/telecom/sms/sms/compose/addPhonebookContact/telecom-sms-sms-compose-addPhonebookContact.html]: \+ `text-center`

File: client/app/telecom/sms/sms/hlr/telecom-sms-sms-hlr.html [hunk 28]
```diff
@@ -331,7 +330,7 @@ <h1 data-translate="sms_sms_hlr_title"></h1>
                         <tbody data-ng-if="SmsHlrCtrl.hlr.isLoading">
                             <tr>
                                 <td colspan="9">
```
Comment (Jisay) [inline client/app/telecom/sms/sms/hlr/telecom-sms-sms-hlr.html]: \+ `text-center`

File: client/app/telecom/sms/sms/incoming/telecom-sms-sms-incoming.html [hunk 29]
```diff
@@ -260,7 +258,7 @@ <h1 data-translate="sms_sms_incoming_title"></h1>
                 <tbody data-ng-if="SmsIncomingCtrl.incoming.isLoading">
                     <tr>
                         <td colspan="5">
```
Comment (Jisay) [inline client/app/telecom/sms/sms/incoming/telecom-sms-sms-incoming.html]: \+ `text-center`

File: client/app/telecom/sms/sms/outgoing/telecom-sms-sms-outgoing.html [hunk 30]
```diff
@@ -270,7 +269,7 @@ <h1 data-translate="sms_sms_outgoing_title"></h1>
                 <tbody data-ng-if="SmsOutgoingCtrl.outgoing.isLoading">
                     <tr>
                         <td colspan="8">
```
Comment (Jisay) [inline client/app/telecom/sms/sms/outgoing/telecom-sms-sms-outgoing.html]: \+ `text-center`

File: client/app/telecom/sms/sms/pending/telecom-sms-sms-pending.html [hunk 31]
```diff
@@ -297,7 +295,7 @@ <h1 data-translate="sms_sms_pending_title"></h1>
                 <tbody data-ng-if="SmsPendingCtrl.pending.isLoading">
                     <tr>
                         <td colspan="8">
```
Comment (Jisay) [inline client/app/telecom/sms/sms/pending/telecom-sms-sms-pending.html]: \+ `text-center`

File: client/app/telecom/freeFax/credit/freeFax-credit.html [hunk 32]
```diff
@@ -22,9 +22,9 @@
                         <span data-ng-if="Credit.cost">
                             {{ 'freefax_credit_unit' | translate:{ quantity: Credit.quantity, fax:Credit.quantity/2 } }} / {{ Credit.cost.withTax.text }} {{:: 'ttc' | translate }}
                         </span>
-                        <oui-loader data-ng-if="Credit.cost === null"
-                                    data-size="s">
-                        </oui-loader>
+                        <div class="text-center" data-ng-if="Credit.cost === null">
```
Comment (AxelPeter) [inline client/app/telecom/freeFax/credit/freeFax-credit.html]: `div` in `span` parent. The centering is needed here ?

File: client/app/telecom/freeFax/faxConfiguration/freeFax-faxConfiguration.html [hunk 33]
```diff
@@ -178,10 +180,9 @@
                                     data-ng-really-window-class="revert-really-buttons"
                                     data-ng-disabled="FaxConfCtrl.generatingPassword">
                             </button>
-                            <oui-loader class="d-inline-block align-middle ml-2"
-                                                 data-ng-if="FaxConfCtrl.generatingPassword"
-                                                 data-size="s">
-                            </oui-loader>
+                            <div class="text-center ml-2" data-ng-if="FaxConfCtrl.generatingPassword">
```
Comment (AxelPeter) [inline client/app/telecom/freeFax/faxConfiguration/freeFax-faxConfiguration.html]: Apparently, the centering is not needed here too.
Before, it was `<oui-loader class="d-inline-block align-middle ml-2" ...`.

File: client/app/telecom/freeFax/faxConfiguration/freeFax-faxConfiguration.html [hunk 34]
```diff
@@ -238,10 +239,9 @@
                                     data-ng-disabled="FaxConfCtrl.loading"
                                     data-translate="cancel">
                             </button>
-                            <oui-loader class="d-inline-block align-middle ml-2"
-                                                 data-ng-if="FaxConfCtrl.loading"
-                                                 data-size="s">
-                            </oui-loader>
+                            <div class="text-center ml-2" data-ng-if="FaxConfCtrl.loading">
```
Comment (AxelPeter) [inline client/app/telecom/freeFax/faxConfiguration/freeFax-faxConfiguration.html]: Same here

File: client/app/telecom/pack/move/address/future/pack-move-address-future.html [hunk 35]
```diff
@@ -6,9 +6,9 @@
             </div>
             <div class="block-emphasis__column">
                 <h3 data-translate="pack_move_line_future_address"></h3>
-                <oui-loader data-ng-if="PackMoveAddressFuture.loading"
-                            data-size="m">
-                </oui-loader>
+	<div class="text-center" data-ng-if="PackMoveAddressFuture.loading">
```
Comment (AxelPeter) [inline client/app/telecom/pack/move/address/future/pack-move-address-future.html]: Bad indentation :)

File: client/app/telecom/pack/slots/domain/activation/pack-domain-activation.html [hunk 36]
```diff
@@ -73,9 +73,9 @@ <h1 data-translate="domain_activation_widget_title"></h1>
                     </div>
                     <div class="mt-2"
                          data-ng-if="toggles.domainLoading">
-                        <oui-loader class="mr-2"
-                                    data-size="s">
-                        </oui-loader>
+	<div class="text-center">
```
Comment (AxelPeter) [inline client/app/telecom/pack/slots/domain/activation/pack-domain-activation.html]: Bad indentation here too

File: client/app/telecom/pack/slots/hubic/activation/pack-hubic-activation.html [hunk 37]
```diff
@@ -14,10 +14,9 @@ <h1 data-translate="hubic_activation_widget-title"></h1>
                  data-translate="hubic_activation_widget-intro">
             </div>
 
-            <oui-loader data-ng-if="PackHubicActivationCtrl.loading"
-                        data-size="m">
-            </oui-loader>
-
+	<div class="text-center" data-ng-if="PackHubicActivationCtrl.loading">
```
Comment (AxelPeter) [inline client/app/telecom/pack/slots/hubic/activation/pack-hubic-activation.html]: Careful, here too

File: client/app/telecom/freeFax/voicemailConfiguration/freeFax-voicemailConfiguration.html [hunk 38]
```diff
@@ -179,10 +179,9 @@
                                 data-ng-disabled="VoicemailConf.loading"
                                 data-translate="cancel">
                             </button>
-                            <oui-loader class="ml-2"
-                                        data-ng-show="VoicemailConf.loading"
-                                        data-size="s">
-                            </oui-loader>
+                            <div class="text-center" data-ng-show="VoicemailConf.loading">
+                                    <oui-spinner data-size="s"></oui-spinner>
```
Comment (Jisay) [inline client/app/telecom/freeFax/voicemailConfiguration/freeFax-voicemailConfiguration.html]: indentation

File: client/app/telecom/overTheBox/configure/overTheBox-configure.html [hunk 39]
```diff
@@ -8,9 +8,9 @@ <h1 data-translate="overTheBox_title"></h1>
 
         <toast-message></toast-message>
 
-        <oui-loader data-ng-show="OverTheBoxConfigure.loading"
-                    data-size="m">
-        </oui-loader>
+        <div class="text-center" data-ng-show="OverTheBoxConfigure.loading">
+                <oui-spinner></oui-spinner>
```
Comment (Jisay) [inline client/app/telecom/overTheBox/configure/overTheBox-configure.html]: indent

File: client/app/telecom/overTheBox/order/order-overTheBox.html [hunk 40]
```diff
@@ -7,9 +7,9 @@ <h1 data-translate="overTheBox_title"></h1>
 
         <toast-message></toast-message>
 
-        <oui-loader data-ng-show="OrderOverTheBox.loaders.checking"
-                    data-size="m">
-        </oui-loader>
+        <div class="text-center" data-ng-show="OrderOverTheBox.loaders.checking">
+                <oui-spinner></oui-spinner>
```
Comment (Jisay) [inline client/app/telecom/overTheBox/order/order-overTheBox.html]: indentation

File: client/app/telecom/overTheBox/order/order-overTheBox.html [hunk 41]
```diff
@@ -39,18 +39,14 @@ <h1 data-translate="overTheBox_title"></h1>
...
                 <span data-translate="order_overTheBox_subscription_request"></span>
             </button>
-
         </div>
 
-        <oui-loader data-ng-show="OrderOverTheBox.loaders.order"
-                    data-size="m">
-        </oui-loader>
+        <div class="text-center" data-ng-show="OrderOverTheBox.loaders.order">
+                <oui-spinner></oui-spinner>
```
Comment (Jisay) [inline client/app/telecom/overTheBox/order/order-overTheBox.html]: indentation

File: client/app/telecom/overTheBox/remote/overTheBox-remote.html [hunk 42]
```diff
@@ -50,10 +50,9 @@
                                 </button>
                             </td>
                             <td>
-                                <oui-loader class="mr-2"
-                                            data-ng-show="remote.status !== 'active'"
-                                            data-size="s">
-                                </oui-loader>
+                                <div class="text-center" data-ng-show="remote.status !== 'active'">
+                                        <oui-spinner data-size="s"></oui-spinner>
```
Comment (Jisay) [inline client/app/telecom/overTheBox/remote/overTheBox-remote.html]: indentation

File: client/app/telecom/overTheBox/remote/overTheBox-remote.html [hunk 43]
```diff
@@ -270,10 +269,9 @@
                                 data-ng-disabled="OverTheBoxRemote.adding || overTheBoxAdd.$invalid"
                                 data-translate="submit">
                             </button>
-                            <oui-loader class="ml-2"
-                                        data-ng-show="FaxConfCtrl.loading"
-                                        data-size="s">
-                            </oui-loader>
+                            <div class="text-center" data-ng-show="FaxConfCtrl.loading">
+                                    <oui-spinner data-size="s"></oui-spinner>
```
Comment (Jisay) [inline client/app/telecom/overTheBox/remote/overTheBox-remote.html]: indentation

File: client/app/telecom/overTheBox/warning/overTheBox-warning-deviceToLink.html [hunk 44]
```diff
@@ -18,9 +18,9 @@ <h3 data-ng-if="DeviceToLink.deviceCount > 1"
               aria-hidden="true"
               data-ng-if="!DeviceToLink.loader">
         </span>
-        <oui-loader data-ng-if="DeviceToLink.loader"
-                    data-size="s">
-        </oui-loader>
+	<div class="text-center" data-ng-if="DeviceToLink.loader">
```
Comment (Jisay) [inline client/app/telecom/overTheBox/warning/overTheBox-warning-deviceToLink.html]: `spinner` must not be centered here. Should stay as default inline-block

File: client/app/telecom/pack/migration/offers/telecom-pack-migration-offers.html [hunk 45]
```diff
@@ -1,9 +1,8 @@
 <div class="telecom-pack-migration-offers">
 
-    <oui-loader data-ng-if="OffersCtrl.loading.init"
-                data-size="m">
-    </oui-loader>
-
+	<div class="text-center" data-ng-if="OffersCtrl.loading.init">
```
Comment (Jisay) [inline client/app/telecom/pack/migration/offers/telecom-pack-migration-offers.html]: indentation

File: client/app/telecom/pack/migration/shipping/telecom-pack-migration-shipping.html [hunk 46]
```diff
@@ -13,10 +13,9 @@
                 <p data-translate="telecom_pack_migration_shipping_explain2"></p>
 
                 <!-- LOADER -->
-                <oui-loader data-ng-if="ShippingCtrl.loading.init"
-                            data-size="m">
-                </oui-loader>
-
+	<div class="text-center" data-ng-if="ShippingCtrl.loading.init">
```
Comment (Jisay) [inline client/app/telecom/pack/migration/shipping/telecom-pack-migration-shipping.html]: take care to the indentation please

File: client/app/telecom/pack/migration/telecom-pack-migration.html [hunk 47]
```diff
@@ -9,10 +9,9 @@ <h1 data-translate="telecom_pack_migration_title"></h1>
     <div class="telecom-section-content">
 
         <!-- LOADER -->
-        <oui-loader data-ng-if="PackMigrationCtrl.loading.init"
-                    data-size="m">
-        </oui-loader>
-
+	<div class="text-center" data-ng-if="PackMigrationCtrl.loading.init">
```
Comment (Jisay) [inline client/app/telecom/pack/migration/telecom-pack-migration.html]: indent

File: client/app/telecom/pack/move/address/current/pack-move-address-current.html [hunk 48]
```diff
@@ -6,9 +6,9 @@
             </div>
             <div class="block-emphasis__column">
                 <h3 data-translate="pack_move_line_current_address"></h3>
-                <oui-loader data-ng-if="PackMoveAddressCurrent.loading"
-                            data-size="m">
-                </oui-loader>
+	<div class="text-center" data-ng-if="PackMoveAddressCurrent.loading">
```
Comment (Jisay) [inline client/app/telecom/pack/move/address/current/pack-move-address-current.html]: indentation

File: client/app/telecom/pack/move/address/future/pack-move-address-future.html (hunk 35 above)
Comment (Jisay) [inline client/app/telecom/pack/move/address/future/pack-move-address-future.html]: indentation

File: client/app/telecom/pack/move/eligibility/address/pack-move-eligibility-address.html [hunk 49]
```diff
@@ -146,10 +146,9 @@
                 data-translate="submit"
                 class="btn btn-primary">
         </button>
-        <oui-loader class="ml-2"
-                    data-ng-if="PackMoveEligibilityAddress.loading"
-                    data-size="s">
-        </oui-loader>
+	<div class="text-center" data-ng-if="PackMoveEligibilityAddress.loading">
```
Comment (Jisay) [inline client/app/telecom/pack/move/eligibility/address/pack-move-eligibility-address.html]: indent

File: client/app/telecom/pack/move/eligibility/lineNumber/pack-move-eligibility-lineNumber.html [hunk 50]
```diff
@@ -31,8 +31,7 @@
             data-translate="submit"
             data-ng-disabled="PackMoveEligibilityPhoneNumber.loading || availabilityForm.phoneNumber.$invalid || PackMoveEligibilityPhoneNumber.method !== 'number'">
     </button>
-    <oui-loader class="ml-2"
-                data-ng-if="PackMoveEligibilityPhoneNumber.loading"
-                data-size="s">
-    </oui-loader>
+	<div class="text-center" data-ng-if="PackMoveEligibilityPhoneNumber.loading">
```
Comment (Jisay) [inline client/app/telecom/pack/move/eligibility/lineNumber/pack-move-eligibility-lineNumber.html]: change tab to spaces

File: client/app/telecom/telephony/line/calls/filtering/telecom-telephony-line-calls-filtering.html [hunk 51]
```diff
@@ -127,16 +125,16 @@ <h1 data-translate="telephony_line_calls_filtering_title"></h1>
...
-                                                data-ng-if="CallsFilteringCtrl.options.isUpdating.absentSubscriber"
-                                                data-size="s">
-                                    </oui-loader>
-                                </span>
-                                <span class="oui-checkbox__icon">
-                                    <i class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></i>
-                                    <i class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></i>
-                                    <i class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></i>
+                                    <span data-i18n-static="telephony_line_calls_filtering_absent"></span>
+                                    <div class="text-center" data-ng-if="CallsFilteringCtrl.options.isUpdating.absentSubscriber">
```
Comment (AxelPeter) [inline client/app/telecom/telephony/line/calls/filtering/telecom-telephony-line-calls-filtering.html]: `div` in `span`

File: client/app/telecom/telephony/line/calls/filtering/telecom-telephony-line-calls-filtering.html [hunk 52]
```diff
@@ -104,15 +102,15 @@ <h1 data-translate="telephony_line_calls_filtering_title"></h1>
...
-                                    <oui-loader class="ml-2"
-                                                data-ng-if="CallsFilteringCtrl.options.isUpdating.doNotDisturb"
-                                                data-size="s">
-                                    </oui-loader>
-                                </span>
-                                <span class="oui-checkbox__icon">
-                                    <i class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></i>
-                                    <i class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></i>
-                                    <i class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></i>
+                                    <div class="text-center" data-ng-if="CallsFilteringCtrl.options.isUpdating.doNotDisturb">
```
Comment (AxelPeter) [inline client/app/telecom/telephony/line/calls/filtering/telecom-telephony-line-calls-filtering.html]: `div` in `span`, here too

File: client/app/telecom/telephony/line/calls/filtering/telecom-telephony-line-calls-filtering.html [hunk 53]
```diff
@@ -80,15 +78,15 @@ <h1 data-translate="telephony_line_calls_filtering_title"></h1>
...
-                                    <oui-loader class="ml-2"
-                                                data-ng-if="CallsFilteringCtrl.options.isUpdating.anonymousCallRejection"
-                                                data-size="s">
-                                    </oui-loader>
-                                </span>
-                                <span class="oui-checkbox__icon">
-                                    <i class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></i>
-                                    <i class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></i>
-                                    <i class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></i>
+                                    <div class="text-center" data-ng-if="CallsFilteringCtrl.options.isUpdating.anonymousCallRejection">
```
Comment (AxelPeter) [inline client/app/telecom/telephony/line/calls/filtering/telecom-telephony-line-calls-filtering.html]: Same here

File: client/app/telecom/telephony/alias/configuration/members/oldPabx/telecom-telephony-alias-configuration-members-oldPabx.html [hunk 54]
```diff
@@ -90,9 +90,9 @@ <h1 data-translate="telephony_alias_configuration_members_title"></h1>
                              data-ng-class="{'pl-5': !$ctrl.agentInEdition && !$ctrl.agentToDelete}">
                             <strong class="member-description my-3 align-middle"
                                     data-ng-class="{'px-3': !$ctrl.memberInEdition && !$ctrl.memberToDelete}">
-                                <oui-loader data-size="s"
-                                            data-ng-if="agent.description === undefined">
-                                </oui-loader>
+	<div class="text-center" data-ng-if="agent.description === undefined">
```
Comment (AxelPeter) [inline client/app/telecom/telephony/alias/configuration/members/oldPabx/telecom-telephony-alias-configuration-members-oldPabx.html]: `div` in `strong`

File: client/app/telecom/pack/xdsl/modem/managedByOvh/pack-xdsl-modem-managedByOvh.html [hunk 55]
```diff
@@ -23,9 +23,9 @@
                     data-ng-really-window-class="revert-really-buttons">
             </span>
             <span data-ng-if="ManagedByCtrl.mediator.tasks.changeModemConfigManagement">
-                <oui-loader class="mr-2"
-                            data-size="s">
-                </oui-loader>
+	<div class="text-center">
```
Comment (AxelPeter) [inline client/app/telecom/pack/xdsl/modem/managedByOvh/pack-xdsl-modem-managedByOvh.html]: `div` in `span`

File: client/app/telecom/pack/xdsl/modem/firewall/pack-xdsl-modem-firewall.html [hunk 56]
```diff
@@ -14,9 +14,9 @@
             type="submit"
             data-ng-disabled="!FirewallCtrl.mediator.capabilities.canChangeEasyFirewallLevel || FirewallCtrl.firewallCurrentLevel === FirewallCtrl.firewallCurrentLevelTmp">
             <span data-ng-if="FirewallCtrl.mediator.tasks.changeModemConfigEasyFirewallLevel">
-                <oui-loader class="mr-2"
-                            data-size="s">
-                </oui-loader>
+	<div class="text-center">
```
Comment (AxelPeter) [inline client/app/telecom/pack/xdsl/modem/firewall/pack-xdsl-modem-firewall.html]: `div` in `button`

File: client/app/telecom/overTheBox/warning/overTheBox-warning-deviceToLink.html (hunk 44 above)
Comment (AxelPeter) [inline client/app/telecom/overTheBox/warning/overTheBox-warning-deviceToLink.html]: `div` in `button`

**Pull request closed without merging**